Wrappers/Python/cil/optimisation/functions/L2NormSquared.py

```diff
@@ -21,72 +21,59 @@
 from cil.framework import DataContainer
 from cil.optimisation.operators import DiagonalOperator
 
+
 class L2NormSquared(Function):
-    
+
     r""" L2NormSquared function: :math:`F(x) = \| x\|^{2}_{2} = \underset{i}{\sum}x_{i}^{2}`
-          
+
     Following cases are considered:
-                
+
         a) :math:`F(x) = \|x\|^{2}_{2}`
         b) :math:`F(x) = \|x - b\|^{2}_{2}`
+
+    Parameters
+    ----------
+    
+    b:`DataContainer`, optional
+        Translation of the function
+                
         
-    .. note::  For case b) case we can use :code:`F = L2NormSquared().centered_at(b)`,
-               see *TranslateFunction*.
-        
-    :Example:
-        
+    Note
+    -----
+    
+    For case b) case we can use :code:`F = L2NormSquared().centered_at(b)`, see *TranslateFunction*.
```

Comment:
Here's a bigger can of worms 🪱 🪱 
Looking at this documentation I noticed that we use 'center' rather than 'centre' in the function methods
This is inconsistent with our:
https://github.com/TomographicImaging/CIL/blob/ff4741092b7f10bde5e0572c0ac103288a163ffc/Wrappers/Python/cil/processors/CentreOfRotationCorrector.py#L25
And we should be consistent across CIL! I'll open an issue...

Reply:
https://github.com/TomographicImaging/CIL/issues/1612

Reply:
Thanks for reviewing @lauramurgatroyd